spec/fabricators/user_fabricator.rb

```diff
@@ -132,3 +132,10 @@
 Fabricator(:unicode_user, from: :user) do
   username { sequence(:username) { |i| "Löwe#{i}" } }
 end
+
+Fabricator(:bot, from: :user) do
+  id do
+    min_id = User.minimum(:id)
+    [(min_id || 0) - 1, -10].min
```

Comment:
Just curious, do you reckon we can do `-Time.now.to_i`? I did something like that recently.